srml/support/src/dispatch.rs

```diff
@@ -481,6 +481,8 @@ macro_rules! decl_module {
 				$origin_type;
 				$from;
 				$fn_vis fn $fn_name (
+					// TODO: do we allow those meta here probbly not but so we have to enforce that
```

Comment:
Every `TODO` should be an issue in the github issue tracker and the issue link should be attached to the comment.

(Besides that, I don't understand this `TODO`)

Reply:
(it was a personnal as other TODO below reminder, sorry)
it actually means that if there was other attribute #[..] in front of a parameter they wouldn't be used in code generation apart from our compact use.
But as it wasn't allowed before and we don't allow any other things. It is completely OK